### PR TITLE
[Refactor][kubectl-plugin] Change kubectl ray cluster get to get cluster

### PR DESCRIPTION
**Pull request by MortalHappiness**
## Why are these changes needed?

See the description in the corresponding issue for details.

## Related issue number

Closes: ray-project/kuberay#2492

## Checks

- [x] I've made sure the tests are passing.
- Testing Strategy
   - [x] Unit tests
   - [x] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: kubectl-plugin/pkg/cmd/get/get.go [hunk 1]
```diff
@@ -8,11 +8,12 @@ import (
...
 )
 
-func NewClusterCommand(streams genericclioptions.IOStreams) *cobra.Command {
+func NewGetCommand(streams genericclioptions.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:          "cluster",
-		Short:        "Manage ray cluster resources",
-		Long:         `Allow users to manage and retrieve ray cluster information and resources.`,
+		Use:          "get",
+		Short:        "Display one or many Ray resources.",
```
Comment (andrewsykim) [inline kubectl-plugin/pkg/cmd/get/get.go:14]: This is technically incorrect since it only supports getting RayCluster right?

**Review by andrewsykim — COMMENTED** [review]
@chiayi PTAL too

File: kubectl-plugin/pkg/cmd/get/get.go (hunk 1 above)
Comment (MortalHappiness) [reply kubectl-plugin/pkg/cmd/get/get.go:14]: Won't we support getting RayJob and RayService in the future?

Comment (andrewsykim) [reply kubectl-plugin/pkg/cmd/get/get.go:14]: ah good point

**Comment by chiayi** [comment]
LGTM!

**Review by kevin85421 — APPROVED** [review]
approve because @chiayi has already approved the PR

**Review by rkooo567 — APPROVED** [review]

**Pull request merged**